runtime/src/bridge.rs

```diff
@@ -365,6 +386,7 @@ mod tests{
 
 	parameter_types! {
 		pub HashId: chainbridge::ResourceId = chainbridge::derive_resource_id(1, &blake2_128(b"hash"));
+		//TODO rename xRAD to xCFG and create new mapping
 		pub NativeTokenId: chainbridge::ResourceId = chainbridge::derive_resource_id(1, &blake2_128(b"xRAD"));
```

Comment:
What is the process to change the resourceID @mikiquantum ?
Does it require another Upgrade?

Reply:
Yes, but that will require updates on bridge mappings, so for now I wouldn't change it and do it when we upgrade bridge code.